src/app/WriteHandler.cpp

```diff
@@ -243,7 +343,16 @@ Status WriteHandler::ProcessWriteRequest(System::PacketBufferHandle && aPayload,
     }
 
     AttributeDataIBsParser.GetReader(&AttributeDataIBsReader);
-    err = ProcessAttributeDataIBs(AttributeDataIBsReader);
+
+    if (mpExchangeCtx != nullptr && mpExchangeCtx->IsGroupExchangeContext())
```

Comment:
Again, TODO comment about why mpExchangeContext can be null here (if it can at all).

Reply:
done